buildkite/graphql/test_suite.graphql

```diff
@@ -0,0 +1,94 @@
+query getTestSuite($id: ID!, $teamCount: Int) {
```

Comment:
This isn't using the `getNode` query because it needs the teams linked to the suite with a dynamic number of tests returned